packages/cli/src/commands/generate/cell/templates/stories.js.template

```diff
@@ -1,14 +1,16 @@
-import type { ComponentStory } from '@storybook/react'
-
 import { Loading, Empty, Failure, Success } from './${pascalName}Cell'
 import { standard } from './${pascalName}Cell.mock'
 
 export const loading = () => {
-  return Loading ? <Loading /> : <></>
+  render: (args) => {
+    return Loading ? <Loading {...args} /> : <></>
+  },
 }
 
 export const empty = () => {
-  return Empty ? <Empty /> : <></>
+  render: (args) => {
+    return Empty ? <Empty {...args} /> : <></>
+  }
 }
 
 export const failure: ComponentStory<typeof Failure> = (args) => {
```

Comment:
@jtoar I'm not sure why there's TypeScript code within the JS file

Reply:
@ahaywood I think the extension gets changed to whatever the project prefers, and transpiled if it's JS. But you're right this should probably end in `.tsx.template` instead of `.js.template`

Reply:
I converted this into .tsx.template

Then, I [found a function called `transformTSToJS`](https://github.com/ahaywood/redwood/blob/981c5012aa4c8da25d2e5e16c9e8f38de74df65c/packages/cli/src/lib/index.js#L243) that strips all the types 🤩